tests/suites/test_suite_arc4.function

```diff
@@ -7,6 +7,26 @@
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS */
```

Comment:
Please also depend on `!MBEDTLS_PARAM_FAILED_ALT`

Reply:
Thanks, I wasn't aware. I will fix it.

Reply:
Fixed.